zerver/migrations/0011_remove_guardian.py

```diff
@@ -9,34 +9,8 @@
 # Translate the UserProfile fields back to the old Guardian model
 def unmigrate_guardian_data(apps, schema_editor):
     # type: (StateApps, DatabaseSchemaEditor) -> None
-    Permission = apps.get_model('auth', 'Permission')
-    ContentType = apps.get_model('contenttypes', 'ContentType')
-    UserProfile = apps.get_model('zerver', 'UserProfile')
-
-    try:
-        administer = Permission.objects.get(codename="administer")
-    except Permission.DoesNotExist:
-        administer = Permission.objects.create(codename="administer")
-    try:
-        api_super_user = Permission.objects.get(codename="api_super_user")
-    except Permission.DoesNotExist:
-        api_super_user = Permission.objects.create(codename="api_super_user")
-    realm_content_type = ContentType.objects.get(app_label="zerver",
-                                                 model="realm")
-
-    # assign_perm isn't usable inside a migration, so we just directly
-    # create the UserObjectPermission objects
-    UserObjectPermission = apps.get_model('guardian', 'UserObjectPermission')
-    for user_profile in UserProfile.objects.filter(is_realm_admin=True):
-        UserObjectPermission(permission=administer,
-                             user_id=user_profile.id,
-                             object_pk=user_profile.realm_id,
-                             content_type=realm_content_type).save()
-    for user_profile in UserProfile.objects.filter(is_api_super_user=True):
-        UserObjectPermission(permission=api_super_user,
-                             user_id=user_profile.id,
-                             object_pk=user_profile.realm_id,
-                             content_type=realm_content_type).save()
+    # Do we really need this?
+    pass
```

Comment:
We do kinda need this migration for users who were previously running an older Zulip server version to eliminate the use of Guardian on those.  The main possible approach for eliminating this is to require that users upgrade to e.g. 1.4.x before they can upgrade to the new 1.5 (or whatever the versions are).  I think it's fine to plan on doing that in the release where we go to Django 1.10.